packages/backend-plugin-api/src/services/helpers/loggerToWinstonLogger.ts

```diff
@@ -26,9 +26,24 @@ class BackstageLoggerTransport extends Transport {
     super(opts);
   }
 
-  log(info: { message: string }, callback: VoidFunction) {
-    // TODO: add support for levels and fields
-    this.backstageLogger.info(info.message);
+  log(info: any, callback: VoidFunction) {
```

Comment:
```suggestion
  log(info: { [name: string]: any }, callback: VoidFunction) {
```

maybe? don't wanna unpack a `null` for example